File: src/main/java/alfheim/client/model/block/ModelSimpleAnyavil.java
package alfheim.client.model.block;

import net.minecraft.client.model.ModelBase;
import net.minecraft.client.model.ModelRenderer;

/**
 * ModelSimpleAnyavil - AlexSocol
 * Created using Tabula 4.1.1
 */
public class ModelSimpleAnyavil extends ModelBase {
	
    public final ModelRenderer shape1;
    public final ModelRenderer shape2;
    public final ModelRenderer shape3;
    public final ModelRenderer shape4;

    public ModelSimpleAnyavil() {
        textureWidth = 52;
        textureHeight = 50;
        shape1 = new ModelRenderer(this, 0, 16);
        shape1.setRotationPoint(-6.0F, 20.0F, -6.0F);
        shape1.addBox(0.0F, 0.0F, 0.0F, 12, 4, 12, 0.0F);
        shape2 = new ModelRenderer(this, 0, 32);
        shape2.setRotationPoint(-5.0F, 19.0F, -4.0F);
        shape2.addBox(0.0F, 0.0F, 0.0F, 10, 1, 8, 0.0F);
        shape3 = new ModelRenderer(this, 0, 41);
        shape3.setRotationPoint(-4.0F, 14.0F, -2.0F);
        shape3.addBox(0.0F, 0.0F, 0.0F, 8, 5, 4, 0.0F);
        shape4 = new ModelRenderer(this, 0, 0);
        shape4.setRotationPoint(-8.0F, 8.0F, -5.0F);
        shape4.addBox(0.0F, 0.0F, 0.0F, 16, 6, 10, 0.0F);
    }

    public void renderAll() {
    	shape1.render(0.0625F);
        shape2.render(0.0625F);
        shape3.render(0.0625F);
        shape4.render(0.0625F);
    }
}

File: src/main/java/alfheim/client/model/block/ModelSimpleAnyavil.kt
package alfheim.client.model.block

import net.minecraft.client.model.*

class ModelSimpleAnyavil: ModelBase() {
	
	val shape1: ModelRenderer
	val shape2: ModelRenderer
	val shape3: ModelRenderer
	val shape4: ModelRenderer
	
	init {
		textureWidth = 52
		textureHeight = 50
		shape1 = ModelRenderer(this, 0, 16)
		shape1.setRotationPoint(-6f, 20f, -6f)
		shape1.addBox(0f, 0f, 0f, 12, 4, 12, 0f)
		shape2 = ModelRenderer(this, 0, 32)
		shape2.setRotationPoint(-5f, 19f, -4f)
		shape2.addBox(0f, 0f, 0f, 10, 1, 8, 0f)
		shape3 = ModelRenderer(this, 0, 41)
		shape3.setRotationPoint(-4f, 14f, -2f)
		shape3.addBox(0f, 0f, 0f, 8, 5, 4, 0f)
		shape4 = ModelRenderer(this, 0, 0)
		shape4.setRotationPoint(-8f, 8f, -5f)
		shape4.addBox(0f, 0f, 0f, 16, 6, 10, 0f)
	}
	
	fun renderAll() {
		shape1.render(0.0625f)
		shape2.render(0.0625f)
		shape3.render(0.0625f)
		shape4.render(0.0625f)
	}
}
